app/models/null_bid.rb

```diff
@@ -0,0 +1,7 @@
+class NullBid
+  def amount
```

Comment:
Yeah let's force it to return a nil and I guess disable Rubocop from being upset by wrapping with a disable